docs/configuration.asciidoc

```diff
@@ -290,19 +290,58 @@ storage use in your Elasticsearch cluster.
 
 |============
 | Environment                            | `Config` key               | Default
-| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `span_frames_min_duration` | `5`
+| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `span_frames_min_duration` | `"5ms"`
 |============
 
-Use this to disable stacktrace collection for spans with a duration shorter than or equal to the given amount of milleseconds.
+Use this to disable stacktrace collection for spans with a duration shorter
+than or equal to the given amount of milleseconds.
 
-The default is 5ms.
+The default is `"5ms"`.
 
 Set it to `-1` to disable the feature and collect stack traces for all spans.
-Setting it to a positive value, e.g. `5`, will limit stack trace collection to spans
-with durations equal or longer than the given value in milliseconds, e.g. 5 milliseconds.
+Setting it to a positive value, e.g. `"5ms"`, will limit stack trace collection
+to spans with durations equal or longer than the given value in milliseconds,
+e.g. 5 milliseconds.
 
 To disable stack trace collection for spans completely, set the value to `0`.
 
+It has to be provided in *<<config-format-duration, duration format>>*.
+
+[float]
+[[config-api-request-time]]
+==== `api_request_time`
+|============
+| Environment                    | `Config` key       | Default
+| `ELASTIC_APM_API_REQUEST_TIME` | `api_request_time` | `"10s"`
+|============
+
+Maximum queue time of the request buffer before sending the request to the
+APM Server.
+A lower value will increase the load on your APM server,
+while a higher value can increase the memory pressure of your app.
+A higher value also impacts the time until data is indexed and searchable in
+Elasticsearch.
+
+This setting is useful to limit memory consumption if you experience a sudden
+spike of traffic.
+It has to be provided in *<<config-format-duration, duration format>>*.
```

Comment:
Description doesn't seem valid any more for v2. This config option now indicates how long an http request is kept open, before a new one is established.

Regarding effects on the server:  The length of the request should not really influence the server's performance any more, except for the overhead of establishing a new http connection. The default timeout on the server is set to 30s, then the connection will be closed.